manifests/cf-manifest/spec/manifest/release_versions_spec.rb

```diff
@@ -20,4 +20,27 @@
         "expected release #{release['name']} version #{release['version']} to have matching version in URL: #{release['url']}"
     end
   end
+
+  specify "manifest versions are newer than the ones in cf_deployment" do
+    def normalise_version(v)
+      Gem::Version.new(v.gsub(/^v/, '').gsub(/^([0-9]+)$/, '0.0.\1'))
+    end
+
+    manifest_releases = manifest_with_defaults.fetch("releases").map { |release|
+      [release['name'], release['version']]
+    }.to_h
+
+    cf_deployment_manifest.fetch("releases").each do |release|
+      next if release['name'].end_with? '-buildpack'
+
+      if manifest_releases.has_key? release['name']
+        cf_deployment_release_version = release.fetch('version')
+        manifest_release_version = manifest_releases[release['name']]
+
+        puts "#{normalise_version(manifest_release_version)} >= #{normalise_version(cf_deployment_release_version)}"
+        expect(normalise_version(manifest_release_version)).to be >= normalise_version(cf_deployment_release_version),
+          "expected release #{release['name']} version #{manifest_release_version} to be older than #{cf_deployment_release_version} as defined in cf-deployment"
```

Comment:
This failure message is backwards. eg it currently outputs something like:
> expected release routing version 0.172.0 to be older than 0.175.0 as defined in cf-deployment

I think it's missing a "not" in there.